src/app.js

```diff
@@ -182,9 +182,6 @@ var run = function () {
   var editor = new Editor(document.getElementById('input'))
 
   // ---------------- FilePanel --------------------
-  // TODO: All FilePanel related CSS should move into file-panel.js
-  // app.js provides file-panel.js with a css selector or a DOM element
-  // and file-panel.js adds its elements (including css), see "Editor" above
   var css = csjs`
     .filepanel-container    {
```

Comment:
You use this `css` on a static container. why not directly put the class there: 
https://github.com/ethereum/browser-solidity/blob/master/index.html#L49 ?